networkit/cpp/centrality/DynApproxBetweenness.cpp

```diff
@@ -7,172 +8,316 @@
  */
 
 #include <networkit/centrality/DynApproxBetweenness.hpp>
+#include <networkit/auxiliary/Log.hpp>
+#include <networkit/auxiliary/NumericTools.hpp>
 #include <networkit/auxiliary/Random.hpp>
+#include <networkit/centrality/Centrality.hpp>
+#include <networkit/components/StronglyConnectedComponents.hpp>
 #include <networkit/distance/Diameter.hpp>
-#include <networkit/graph/GraphTools.hpp>
-#include <networkit/distance/DynDijkstra.hpp>
 #include <networkit/distance/DynBFS.hpp>
-#include <networkit/auxiliary/Log.hpp>
-#include <networkit/auxiliary/NumericTools.hpp>
-
+#include <networkit/distance/DynDijkstra.hpp>
+#include <networkit/graph/GraphTools.hpp>
 
+// networkit-format
```

Comment:
We are not using `networkit-format` anymore. Classes that do not comply with the coding style are marked with `no-networkit-format`.